sdk/appcenter-identity/src/main/java/com/microsoft/appcenter/identity/Identity.java

```diff
@@ -29,6 +29,7 @@
 import com.microsoft.appcenter.utils.async.DefaultAppCenterFuture;
 import com.microsoft.appcenter.utils.context.AbstractTokenContextListener;
 import com.microsoft.appcenter.utils.context.AuthTokenContext;
+import com.microsoft.appcenter.utils.context.UserInformation;
```

Comment:
Optimize imports, it is suspicious that an import is added and not used.

Reply:
Will be fixed by rollbacking https://github.com/Microsoft/AppCenter-SDK-Android/pull/1047#discussion_r273196308